### ISSUE TITLE
Port themes to hubzilla?

### ISSUE THREAD

**Issue by kenrestivo** [DeadSuperHero/hubzilla-themes#9]
Looks like a config.php is required for each theme in order for these themes to be ported in hubzilla.

Is that all that's required or is it a huge job? Anyone working on it?

https://hub.spaz.org/display/1c83b57f38aec1a155e01a668e09f5c87d6195d6a235e35cf27734ea74e58c76@hub.spaz.org?zid=bamfic%40hub.spaz.org

**Comment by kenrestivo** [comment]
Update: looks like it's not, and they're usable without a config.php file at all.

They just need to be copied into the directory and permitted in the admin screen.

**Comment by DeadSuperHero** [comment]
If it helps, I would be happy to make sure all of my themes are compatible.

Out of curiosity, is Hubzilla the main development tree now? I'm currently running Red on my hub, and am eager to switch over.

**Comment by kenrestivo** [comment]
Hubzilla seems like where all the development action is these days, it's the follow-on to red.

We're running it, and chasing HEAD on git daily, and the team seems very diligent about not committing breaking changes, so it's been very solid so far.

I was able to get the themes to run in hubzilla without a config.php file, but I had to manually add them to the list of allowed themes using the config command line, and also it seems like the user override of CSS isn't working on the red themes.

They're great themes; we're using stumble as the default at the moment.
